components/higher-order/with-focus-outside/test/index.js

```diff
@@ -33,55 +36,70 @@ describe( 'withFocusOutside', () => {
 		}
 	);
 
-	it( 'should not call handler if focus shifts to element within component', () => {
-		const callback = jest.fn();
-		const wrapper = mount( <EnhancedComponent onFocusOutside={ callback } /> );
+	// this is needed because TestUtils does not accept a stateless component.
+	// anything run through a HOC ends up as a stateless component.
+	const getTestComponent = ( WrappedComponent, props ) => {
+		class TestComponent extends Component {
+			render() {
+				return <WrappedComponent { ...props } />;
+			}
+		}
+		return <TestComponent />;
+	};
+
+	const simulateEvent = ( event, position = 0 ) => {
```

Comment:
`position` is confusing in this context. I assumed this is position on the screen at first. Did you mean `index`?

Reply:
change implemented